string robot_status
string robot_state
int64 room_id
string room_name
float32 battery_level